### ISSUE TITLE
No playbar showing

### ISSUE THREAD

**Issue by Kai0ken** [yuliskov/SmartTubeNext#558]
Hello :), 

I am comming from "Smart YouTube TV" since it is not working anymore (no play possible). Yesterday I downloaded the "SmartTubeNext" app for the first time ("STubeNext_storig_v10.135_r.apk" to be exact). I really like the design of "Smart YouTube TV", but "SmartTubeNext" is really much faster. Thank you for that. 
But unfortunately on the "SmartTubeNext" app there is no playbar or anything like that showing when I pause a video (with "Smart YouTube TV" it showed a lot of information when paused). For "SmartTubeNext" it shows "nothing" except the video when "paused". Could this please be fixed? I want to see the details, e.g. resolution or playtime "XX:XX/YY:YY" (example: 01:24/02:05) so I can see of many time left of a video and how long I have watched (only when clicked "pause"). 
Thank you in advance. Keep up the good work. Nice and fast(er) app!
Have a nice weekend.

PS: Or is this maybe already possible and I just could not find it?

**Comment by foreveryoungforty** [comment]
Hello, 135 was an experimental release. I advise you to uninstall it and to install 141 via Latest release link on main page (https://github.com/yuliskov/SmartYouTubeTV/releases/download/beta/smarttube_beta.apk). 
Please explore the settings to learn about all possibilities of the app and the player and the visualisation of the playbar, ok-button and so on.... you will be surprised what is already possible :-)

**Comment by Kai0ken** [comment]
Thanks for your reply. Now I have updated to 141, but unfortunately this problem still exists. If I rewind it shows the playbar, but not if I only press pause. Hope this gets fixed soon.

**Comment by foreveryoungforty** [comment]
You can do so by setting of the ok button in video player to show player and pauze. 
This is an extra option for users who want it that way. 
There are many users who absolutely don't want the player ui while pausing, because they want a clear paused picture without buttons and info.
Tastes and preferences differ among users. Therefore you can customize your preference of pauzing by the setting of the ok-button. Hope this helps you.

**Comment by Kai0ken** [comment]
You are correct, "ok button" does show the playbar. But is it possible to change/switch it to the "pause button" of the FireTV Stick like it was in the "Smart YouTube TV" app? Maybe as option in the settings like "switch ok button with pause button" or anything like that.

**Comment by foreveryoungforty** [comment]
I think we best leave the extra functionality of the OK-button as it is, as it was also that way in SYTV.
I think the best and easiest solution for your request is an optional setting in the video player functions to show or not show the player ui when using the common media buttons (play, pauze, stop, prev, next, rew and ffw) on remote control, so the user is free to choose what he likes.

**Comment by Kai0ken** [comment]
Yes, okay.

**Comment by ACR-Jeff** [comment]
Modifying the OK (D-Pad) button can cause you other issues, This may end up being a bigger issue then. 
What you can do is use your `D-pad and arrow down` when in a video, It should then show the Player UI in current releases, 
You then can use the Ok button for play and pause when you have `Settings > Ok button behavior > Only Pause or UI and pause` selected 
At least this is the way it works on all of my androidtv boxes today.
